backend/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": [
+    {
+      "name": "spring.security.ignored",
+      "type": "java.lang.String",
+      "description": "Description for spring.security.ignored."
+  }
+] }
```

Comment:
Thank you very much for a contribution! 

Could you only please tell me why this file has been added?

Reply:
There is no need for this file. I didn't want to add so I've removed it. 
It's metadata for spring configuration more info [here](https://docs.spring.io/spring-boot/docs/current/reference/html/configuration-metadata.html).

Reply:
Now I remember why I added this. Intellij was complaining about the property on yaml. It's just a cosmetic warning. It seems this spring property was missing for intellij  so the only way to skip the warning was to explicitly add the metadata info.

Reply:
All right, thanks for clarifying it. 

And once again - great job! Thanks for contributing.